### DISCUSSION TITLE
Allow groups to be collapsed in MultiSelect

### DISCUSSION THREAD

**Discussion by aprilmintacpineda** [mantinedev/mantine#2316] [Feature requests]
Add a prop to `MultiSelect` `collapsible` that when `true` will allow groups to be collapsed, as in like using `Collapse` component. This will allow users to select the value they want to select faster when there are a few groups on the component.

**Comment by rtivital** [comment]
I guess you just want a tree select component – https://github.com/react-component/tree-select
It is in the backlog, but it is not planned to be released until 6.0 is out.

**Reply by aprilmintacpineda** [reply]
Looking at the demo, it does seem to do that, only it has a different behaviour and structure/UI from MultiSelect
